RFC.md

```diff
@@ -1,15 +1,14 @@
 # K-Nearest Neighbors(KNN) Search Plugin
-
-The purpose of this document is to request the community for comments (RFC) for a new Open Distro for Elasticsearch KNN plugin and to collect feedback from the community. This RFC covers the high-level functionality of the KNN plugin and does not cover architecture and implementation details.
+The purpose of this document is to introduce a new Open Distro for Elasticsearch KNN plugin and to request comments from the community (RFC). This RFC covers only the high-level functionality of the KNN plugin and does not cover architecture and implementation details.
 
 ## Problem Statement
-Customers with similarity-based search use cases such as face search, fraud detection and recommendation systems have a need for finding K-Nearest Neighbor (KNN) using the machine learning models. The existing solutions for KNN search were inefficient and did not operate well at scale. Brute-force KNN search using CPU is very computationally intensive and generally too slow for large-scale real-time search applications.
+Customers with similarity-based search use cases like facial recognition, fraud detection and recommendation systems have a need for finding the K-Nearest Neighbor (KNN) of vectors in high dimensional space. The existing solutions for KNN search in distributed systems either do not operate well at scale or are difficult to use and integrate with current systems.
```

Comment:
"The existing solutions for KNN search do not operate well at scale, difficult to use, computationally intensive and generally too slow for large-scale real-time search applications." ?

Reply:
Thanks addressed this